app/assets/javascripts/components/cookie_banner.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
+  var seenCookieBannerName = "seen_cookie_banner";
+  var seenCookieBanner = getCookie(seenCookieBannerName);
+  var cookieBanner = document.querySelector("#global-cookie-message");
+
+  if (!seenCookieBanner) {
+    cookieBanner.classList.add("govuk-cookie-banner--visible");
+    setCookie(seenCookieBannerName, 1, 365);
```

Comment:
Is it ok that we only show this to users once? Do we need some sort of "yes, I read this" action to clear it?

Reply:
I'm not sure. I can try to find out.

The now deprecated GOV.UK Template that the Service manual refers to [used a similar method](https://github.com/alphagov/govuk_template/blob/cde91582d0a3bc67d5df1c9484de558548361247/source/assets/javascripts/cookie-bar.js) of just showing the cookie banner on first visit so there will be dozens of services doing the same.

Reply:
I have changed the banner to show every 90 days, so if you revisit after submitting you shouldn't see it, but its frequent enough that is should catch cookie policy amends.

I think the part about having an accept button, we look at as a separate dev.